### DISCUSSION TITLE
在下载标签页里下载mod时可以选择筛选出以某个mod为前置的所有mod

### DISCUSSION THREAD

**Discussion by Chaos2233** [Meloong-Git/PCL#1232] [提问] [answered]
我经常用PCL下模组，如果有这个功能在下载一些扩展模组多的模组（比如机械动力宇宙）时会方便很多

**Comment by Chaos2233** [comment]
抱歉我好像错版了 请问功能投票在哪里发起啊

**Comment by zkitefly** [comment]
新功能建议请到 [此处](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml) 进行反馈

在龙猫决定要做的时候，issue 就会转到 discussions 这进行投票的哦

**Comment by WForst-Breeze** [comment, marked as answer]
和 #953 类似，很抱歉此功能不考虑制作，谢谢qwq
